wrappers/IWearLogger/src/IWearLogger.cpp

```diff
@@ -141,6 +185,7 @@ class IWearLogger::impl
         virtualSphericalJointKinSensors;
 
     std::unordered_map<std::string, std::string> wearable2MatlabNameLookup;
+    std::unordered_map<std::string, std::unique_ptr<yarp::os::BufferedPort<yarp::os::Bottle>>> wearable2YarpPortLookup;
```

Comment:
Could you make a `using` statement to clarify what the key is?

Something like:

```suggestion
    // Note: I'm not sure the following is correct
    using WearableSensorName = std::string;
    using MatlabChannelName = std::string;

    std::unordered_map<WearableSensorName, MatlabChannelName> wearable2MatlabNameLookup;
    std::unordered_map<WearableSensorName, std::unique_ptr<yarp::os::BufferedPort<yarp::os::Bottle>>> wearable2YarpPortLookup;
```

Reply:
Done in https://github.com/robotology/wearables/pull/123/commits/afacfdf1c362f7bf87ef28f853f25037a6a3ca8a